examples/cmd/encrypt.go

```diff
@@ -99,15 +99,19 @@ func encrypt(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	baseKasUrl := platformEndpoint
+	if !strings.HasPrefix(baseKasUrl, "http://") && !strings.HasPrefix(baseKasUrl, "https://") {
+		baseKasUrl = fmt.Sprintf("http://%s", baseKasUrl)
```

Comment:
```suggestion
		baseKasUrl = fmt.Sprintf("https://%s", baseKasUrl)
```

Reply:
We should require http to be explicit, and default https

Reply:
this is just for the example which we run for testing and development locally (usually without tls)

Reply:
As long as the example wont be referenced for future implementations, I am ok with that.